elliott/elliottlib/cli/find_bugs_sweep_cli.py

```diff
@@ -376,25 +381,22 @@ def categorize_bugs_by_type(bugs: List[Bug], advisory_id_map: Dict[str, int],
                 logger.info(f"{kind} build found for #{bug.id}, {package_name} ")
                 found.add(bug)
                 bugs_by_type[kind].add(bug)
+            elif bug.id in permitted_bug_ids:
+                logger.info(f'Including tracker bug #{bug.id} since it is explicitly included in the assembly config')
+                found.add(bug)
+                bugs_by_type['image'].add(bug)
```

Comment:
Even if we add the tracker to the image advisory, later `elliott attach-cve-flaws` and `elliott verify-bugs` runs will also fail due to missing build. Maybe it is more reasonable to raise an error when this occurs?